README.md

```diff
@@ -218,6 +218,30 @@ yarn hardhat divvi-registry:upgrade \
     --proxy-address 0x2f5E320698dB89CbefB810Fa19264103d99aAFB1
 ```
 
+To upgrade an existingRewardPoolFactory from V1 to V2 (with protocol fee support), run:
```

Comment:
The full process here also requires us to deploy a new `RewardPool` implementation and call `setImplementation` on the `RewardPoolFactory` to point to the new implementation contract.

Reply:
I suppose that we could add the new implementation address as an (optional?) argument to the `initializeV2` method as well, to solve both problems at once?

Reply:
I'm planning on upgrading all our RewardPoolFactories today, so I've added this feature in!